src/transformers/utils/import_utils.py

```diff
@@ -68,7 +68,7 @@ def _is_package_available(pkg_name: str, return_version: bool = False) -> Union[
 # This is the version of torch required to run torch.fx features and torch.onnx with dictionary inputs.
 TORCH_FX_REQUIRED_VERSION = version.parse("1.10")
 
-ACCELERATE_MIN_VERSION = "0.21.0"
+ACCELERATE_MIN_VERSION = "0.26.0"
```

Comment:
I think it's fine to leave it as 0.21.0 and maybe just add a warning for users that are trying to do FSDP + QLoRA redirecting them to upgrade accelerate if the version is below 0.26?

Reply:
Warning or explicit err